Assets/Scripts/Control/SceneChanger.cs

```diff
@@ -44,6 +44,8 @@ private static void LoadFlyingScene()
             
             GameManager.Instance.InGameButtons.SetActive(false);
             GameManager.Instance.ItemInventory.SetActive(false);
+
+            GameManager.Instance.IncrementLevel();
```

Comment:
Des würd ich eher in `GameManager.GameOver()` machen. So wird ja das erste Level quasi übersprungen